tck/src/main/java/org/eclipse/microprofile/concurrency/tck/ManagedExecutorTest.java

```diff
@@ -1230,6 +1230,65 @@ public void shutdownPreventsAdditionalSubmits() throws ExecutionException, Inter
         }
     }
 
+    /**
+     * Verify that the ManagedExecutor.Builder can be used to create multiple ManagedExecutors with 
+     * different configured contexts.
+     */
+    @Test
+    public void reuseManagedExecutorBuilder() throws ExecutionException, InterruptedException, TimeoutException {
+        ManagedExecutor.Builder builder = ManagedExecutor.builder()
+                .propagated()
+                .cleared(Buffer.CONTEXT_NAME);
+        
+        ManagedExecutor clearingExecutor = builder.build();
+
+        builder.propagated(Buffer.CONTEXT_NAME);
+        try {
+            builder.build();
+            Assert.fail("ManagedExecutor.Builder.build() should throw an IllegalStateException for set overlap between propagated and cleared");
```

Comment:
I think this is already part of another test dedicated to it.
It also looks weird to first change the state to something you know should blow up and then still use the same instance (which you deliberately shifted to "broken state") to build something valid.